test/e2e/bindata_util.go

```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package e2e
 
 import (
-	"os"
-	"path/filepath"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/generated"
+	"os"
+	"path/filepath"
```

Comment:
this ordering will be cleaned up once i push the final rebase.  letting it sit for now while e2e's run...